src/fragments/start/sample-apps/android/todo-app.mdx

```diff
@@ -101,4 +101,4 @@ These features are implemented using Amplify DataStore.
 
 ## UI Experience
 
-![](/images/todo-app.png)
+![Shows screen by screen workflow of how Android Sample App would look like](/images/todo-app.png)
```

Comment:
For this image, since we have some text describing the different screens, do you think it would make sense to also list it out in text form as list?

We could have a section for the "initial screen" workflow and one section for the "toolbar" workflow.

One thing to consider is that if the content from this image isn't necessary to understand the app features of the sample app then I think what you have now is okay.

Reply:
We have excessive documentation in the sample app repo to explain the different screens and the workflow and hence it is not important to explain them here. Keeping it as is.